go.sum

```diff
@@ -83,6 +83,7 @@ github.com/docker/docker v0.7.3-0.20190327010347-be7ac8be2ae0/go.mod h1:eEKB0N0r
 github.com/docker/go-units v0.3.3/go.mod h1:fgPhTUdO+D/Jk86RDLlptpiXQzgHJF7gydDDbaIK4Dk=
 github.com/docker/spdystream v0.0.0-20160310174837-449fdfce4d96 h1:cenwrSVm+Z7QLSV/BsnenAOcDXdX4cMv4wP0B/5QbPg=
 github.com/docker/spdystream v0.0.0-20160310174837-449fdfce4d96/go.mod h1:Qh8CwZgvJUkLughtfhJv5dyTYa91l1fOUCrgjqmcifM=
+github.com/docopt/docopt-go v0.0.0-20180111231733-ee0de3bc6815 h1:bWDMxwH3px2JBh6AyO7hdCn/PkvCZXii8TGj7sbtEbQ=
```

Comment:
any idea why `go.sum` is changing when 0 go files were changed?

Reply:
I am guessing it has to do with my local setup. VSCode keeps auto-updating my `go.mod` and `go.sum` in weird ways. However, I am not sure how this would have passed CI if `go mod tidy` changes anything. I will troubleshoot as I update this PR with the non-forked version of `build` :+1: